entrypoint.bash

```diff
@@ -4,24 +4,24 @@ set -o errexit -o pipefail
 
 sudo chown builduser:builduser /opt/build/
 
-export DEVICE=bluejay
-export BUILD_ID=TQ2A.230505.002
-export OFFICIAL_BUILD=true
-export BUILD_DATETIME=1686159583
-export BUILD_NUMBER=2023060700
+export DEVICE=${DEVICE:=bluejay}
+export BUILD_ID=${BUILD_ID:="TQ2A.230505.002"}
+export OFFICIAL_BUILD=${OFFICIAL_BUILD:=true}
+export BUILD_DATETIME=${BUILD_DATETIME:=1686159583}
+export BUILD_NUMBER=${BUILD_NUMBER:=2023060700}
 
 echo "[INFO] Downloading and verifying manifest"
 mkdir -p /opt/build/grapheneos
 cd /opt/build/grapheneos
-repo init -u https://github.com/GrapheneOS/platform_manifest.git -b refs/tags/TQ2A.230505.002.2023060700
+repo init -u https://github.com/GrapheneOS/platform_manifest.git -b refs/tags/${BUILD_ID}.${BUILD_NUMBER}
 mkdir ~/.ssh && curl https://grapheneos.org/allowed_signers > ~/.ssh/grapheneos_allowed_signers
 cd .repo/manifests
 git config gpg.ssh.allowedSignersFile ~/.ssh/grapheneos_allowed_signers
 git verify-tag $(git describe)
 cd ../..
 
 echo "[INFO] Syncing GrapheneOS tree"
-repo sync -j16
+repo sync -j$(nproc)
```

Comment:
I would not advise doing this, as on faster machines this can hit severe rate limits. It should be configurable but can fallback to this.

Reply:
agreed, will be fixed in @Reflux0301' PR